src/compas/geometry/primitives/frame.py

```diff
@@ -175,9 +175,9 @@ def yaxis(self):
     def yaxis(self, vector):
         yaxis = Vector(*vector)
         yaxis.unitize()
-        zaxis = Vector.cross(self.xaxis, yaxis)
+        zaxis = Vector(*cross_vectors(self.xaxis, yaxis))
```

Comment:
`self.axis` is a `Vector`.
we could leave as-is, or change to `self.xaxis.cross(yaxis)`